index.html

```diff
@@ -66,5 +66,148 @@ <h2 class="lets-connect">LET'S CONNECT</h2>
+      </div>
+      <div class="project-three-info">
+      <h3>Tonic</h3>
+      <ul class="project-details-three">
+          <li>Canopy</li>
+          <li>BackEnd Dev</li>
+          <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div>
+      <ul class="languages-works-three">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+    </div>
+      <h6 class="project-link-three">
+        See Project
+      </h6>
+      </div>
+    </div>
+    <div class="works-four">
+      <div class="works-top-four">
+      <a href="#" class="snapshoot-four"><img src="./images/SnapshootPortfolio3.svg" alt="portfolio-four"></a>
+      </div>
+      <div class="project-four-info">
+      <h3>Multi-Post Stories</h3>
+      <ul class="project-details-four">
+          <li>Canopy</li>
+          <li>BackEnd Dev</li>
+          <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div>
+      <ul class="languages-works-four">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+    </div>
+      <h6 class="project-link-four">
+        See Project
+      </h6>
+      </div>
+    </div>
+    </section>
+    <section>
+      <div class="about-me">
+        <div>
+          <h2>About Myself</h2>
+        <p>I'm a software developer! I can help you build a product , feature or website <br> 
+          Look through some of my work and experience. <br> If you like what you see and have a project you need coded, don’t hestiate to contact me!
+        </p>
+      </div>
+        <div class="lets-connect-two">LET'S CONNECT</div>
+        <ul class="social-media-links">
+          <li> <a href="https://twitter.com/SharonVictor16_" target="_blank" rel="noopener" class="social-media-link"><img src="./images/twitter-icon.svg" alt="Twitter">
+              </a>
+          </li>
+          <li> <a href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/" target="_blank" rel="noopener"><img src="./images/linkedi-icon.svg" alt="LinkedIn">
+              </a> 
+          </li>
+          <li> <a href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/" target="_blank" rel="noopener"><img src="./images/github.svg" alt="LinkedIn">
+          </a> 
+          </li>
+        <li> <a href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/" target="_blank" rel="noopener"><img src="./images/medium.svg" alt="LinkedIn">
+        </a> 
+        </li>
+        <li> <a href="https://github.com/sharon-odhiambo" target="_blank" rel="noopener" class="social-media-link"><img src="./images/angelist.svg" alt="Github">
+        </a>
+        </li>
+        </ul>
+        <div>
+          <a href="#" class="get-resume"><img src="./images/resume.png" alt="get-resume"></a>
```

Comment:
- [ ] As required by the previous reviewer, you are supposed to use an HTML <button> and add hover and press. Take a look at the previous image.